modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/api.js

```diff
@@ -41,4 +41,42 @@ const getHighPriorityContacts = async (filter) => {
 	return response.json();
 };
 
-export {getHighPriorityContacts, fetchHeadless};
+const getTicketAttachments = async (search) => {
+	// eslint-disable-next-line @liferay/portal/no-global-fetch
+	const response = await fetch(
+		`${HEADLESS_BASE_URL}${`c/ticketattachments/?search=${search}`}`,
+		{
+			headers: {
+				'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
+				'Cache-Control': 'max-age=30, stale-while-revalidate=30',
+				'x-csrf-token': Liferay.authToken,
+			},
+		}
+	);
+
+	return response;
+};
```

Comment:
We can simplify a lot that way

```suggestion
const getTicketAttachments = async (search) => {
	return fetcher(`c/ticketattachments/?search=${search}`);
};
```

The same works for the other examples on this file, but it's good to double check if will work as expected.

Important that fetcher already parses the `response.json` to when you call `getTicketAttachments` with response.json() is no longer needed